.github/workflows/feature_request.yml

```
@@ -1,7 +1,7 @@
 name: ✨ Feature Request
-description: Suggest a new feature or enhancement for GoCat
+description: Suggest a new feature or ment for GoCat
```

actionlint (1.7.7)

unexpected key "description" for "workflow" section. expected one of "concurrency", "defaults", "env", "jobs", "name", "on", "permissions", "run-name"

(syntax-check)